build.gradle

```diff
@@ -79,7 +79,7 @@ dependencies {
     metalava 'com.android.tools.lint:lint-checks:26.1.0-beta1'
     metalava 'com.android.tools.lint:lint-gradle:26.1.0-beta1'
     metalava 'com.android.tools.lint:lint:26.1.0-beta1'
-    metalava 'org.jetbrains.kotlin:kotlin-stdlib:1.2.21'
+    metalava "org.jetbrains.kotlin:kotlin-stdlib:$versions.kotlin"
```

Comment:
This was missing the `$versions.kotlin` constant.

Reply:
This version is specifically pinned to the version metalava uses.